docs/conf.py

```diff
@@ -74,5 +74,5 @@ def _patched_sphinx_jsonschema_simpletype(self, schema):
 
 
 sjs_wide_format = importlib.import_module("sphinx-jsonschema.wide_format")
-_original_sphinx_jsonschema_simpletype = sjs_wide_format.WideFormat._simpletype
-sjs_wide_format.WideFormat._simpletype = _patched_sphinx_jsonschema_simpletype
+_original_sphinx_jsonschema_simpletype = sjs_wide_format.WideFormat._simpletype  # type: ignore # noqa: PGH003
+sjs_wide_format.WideFormat._simpletype = _patched_sphinx_jsonschema_simpletype  # type: ignore # noqa: PGH003
```

Comment:
`type: ignore` was originally here, and Ruff didn't like it. I'm not sure what kind of type we should be ignoring, so I can't make Ruff happy. Hence the hideous-looking double-ignore on these lines.